src/generic-swap-transaction-builder.ts

```diff
@@ -674,6 +674,7 @@ export class GenericSwapTransactionBuilder {
     priceRoute: OptimalRate,
     dexExchangeParams: DexExchangeParam[],
     maybeWethCallData?: DepositWithdrawReturn,
+    skipApproval?: boolean,
```

Comment:
This param is never passed, looks redundant

Reply:
fixed